librz/bin/p/bin_bf.c

```diff
@@ -19,7 +19,7 @@ static ut64 baddr(RzBinFile *bf) {
 }
 
 static RzList /*<RzBinString *>*/ *strings(RzBinFile *bf) {
-	return NULL;
+	return rz_list_newf((RzListFree)rz_bin_string_free);
```

Comment:
what about this?

Reply:
bf cannot have strings, because the code itself is a string.